apis/core/lib/handlers/csv-source.ts

```diff
@@ -75,7 +75,8 @@ const getCSVSource: express.RequestHandler = asyncMiddleware(async (req, res, ne
     return next(new Error('s3 key not found'))
   }
 
-  res.redirect(directDownload)
+  res.header('Location', directDownload)
```

Comment:
@tpluscode I run into CORS-Problems using redirect and still wondering if there is a better way than this.